lib/ancestry/instance_methods.rb

```diff
@@ -255,7 +255,7 @@ def sibling_ids
     end
 
     def has_siblings?
-      siblings.count > 1
+      siblings.where.not(self.class.primary_key => id).exists?
```

Comment:
future request: wish there were a way to cache this somehow so we could fetch this answer in a single query. Along with has_children?